webapp/src/Controller/API/JudgehostController.php

```diff
@@ -1111,48 +1121,6 @@ private function maybeUpdateActiveJudging(Judging $judging): void
         }
     }
 
-    private function getSubmissionsToJudge(Judgehost $judgehost, $restrictJudgingOnSameJudgehost)
```

Comment:
This is just dead code unrelated to this change? If so, please delete in a separate commit.